RELEASING.adoc

```diff
@@ -1,80 +1,210 @@
 = Releasing Kiali
 
-This document gives the steps required to perform a release of Kiali. This document is only useful to Kiali release managers.
+This document covers the specifics of releasing Kiali, and is meant as a guide
+for Kiali release managers.
+
+The summary of the process is the following:
+
+* Create a new JIRA issue for the release
+* Make sure Getting Started docs are complete
+* Update Kiali UI version
+* Publish the API Docs
+* Update Makefile
+* Create the Release PR
+* Tag Git Repo
```

Comment:
how to tag/tagging strategy for numbers?

Reply:
It's on the section for it, this is just a summary.